contrib/win-installer/test-installer.ps1

```diff
@@ -53,19 +50,15 @@ function Install-Podman {
         [ValidateScript({Test-Path $_ -PathType Leaf})]
         [string]$setupExePath
     )
-    if ($installWSL) {$wslCheckboxVar = "1"} else {$wslCheckboxVar = "0"}
-    if ($installHyperV) {$hypervCheckboxVar = "1"} else {$hypervCheckboxVar = "0"}
     if ($skipWinVersionCheck) {$allowOldWinVar = "1"} else {$allowOldWinVar = "0"}
     if ($skipConfigFileCreation) {$skipConfigFileCreationVar = "1"} else {$skipConfigFileCreationVar = "0"}
 
     Write-Host "Running the installer ($setupExePath)..."
-    Write-Host "(provider=`"$provider`", WSLCheckbox=`"$wslCheckboxVar`", HyperVCheckbox=`"$hypervCheckboxVar`", AllowOldWin=`"$allowOldWinVar`", SkipConfigFileCreation=`"$skipConfigFileCreationVar`")"
+    Write-Host "(provider=`"$provider`", AllowOldWin=`"$allowOldWinVar`", SkipConfigFileCreation=`"$skipConfigFileCreationVar`")"
     $ret = Start-Process -Wait `
                             -PassThru "$setupExePath" `
                             -ArgumentList "/install /quiet `
                                 MachineProvider=${provider} `
-                                WSLCheckbox=${wslCheckboxVar} `
-                                HyperVCheckbox=${hypervCheckboxVar} `
```

Comment:
If I pass this on the cli will the installer error out?
On the end hand it makes sense so cli users know we removed the feature on the other hand this will beak all scripts who use it.

Reply:
It doesn't error out. Microsoft installers ignore parameters that they cannot recognize. That's a bad design for today's standards, but that's how they work. So CLI users won't figure out anything until they run `podman machine init`. Now, `podman machine init` can install WSL automatically, which may make things smoother. But there is a [bug](https://github.com/containers/podman/issues/25234), and most importantly, there are no CI tests, so these things get broken easily. We may want to talk about that more.

Reply:
Yeah I think no error is might better here if we do not want break some scripts, that said they likely find out the hard way if WSL/hyperV is no longer installed.

I guess the only question is if we want to ship "breaking" changes in 5.X or wait until a 6.0. Normally I would be against any breaking changes but this is windows and I don't really care about the installer so if this helps you/us to prevent future problems then I am cool with it.